### ISSUE TITLE
[sklearn#34648] DOC: Array API - mention torch XPU is a bit unstable

### ISSUE THREAD

**Issue by godofecht** [flooooooooooow/flow#302]
**Mirrored from [scikit-learn#34648](https://github.com/scikit-learn/scikit-learn/issues/34648)**

Original title: DOC: Array API - mention torch XPU is a bit unstable
Original labels: Documentation
Created: 2026-08-04T17:19:50Z

---

### Describe the issue linked to the documentation

While running tests and benchmarks with scikit-learn Array API backed by torch XPU, I got many errors:
- float precision (fixed in 2.12)
- JIT hangs (fixed in 2.13)
- And other kind of hang and a very bad slowdown for some matmul: https://github.com/intel/torch-xpu-ops/issues/4805 (hang fixed in upcoming 2.14, but not the slowdown).

I got none of such errors with dpnp backed tests/runs.

### Suggest a potential alternative/fix

At least mention torch XPU is still a bit unstable and that users are encouraged to use the latest versions (fixes are being made quickly).

And maybe mention that dpnp is more robust as of august 2026. Though it might also be slower, I'll have some benchmark results soon.

This is not urgent to fix, I'm opening this issue to keep track of things. We can wait for some benchmark results before editing the array API doc.

---

## Flow context

This issue is mirrored from scikit-learn to explore whether Flow encounters the same problem or can approach it differently. Add analysis below.

**Comment by godofecht** [comment]
Addressed in upstream PR(s). See scikit-learn/scikit-learn PRs for #34648.
